.prettierignore

```diff
@@ -34,3 +34,4 @@ venv/
 *.md
 .github/**/*.yml
 .golangci.yml
+.solhint.json
```

Comment:
```suggestion
contracts/.solhint.json
```
Not positive this is necessary, could run in the root of each package, but it seems more intuitive this way.